docs/administration-guide/01-managing-databases.md

```diff
@@ -15,7 +15,7 @@ Now you’ll see a list of your databases. To connect another database to Metaba
 - Google Analytics
 - [Google BigQuery](databases/bigquery.md)
 - H2
-- [MongoDB (version 3.4 or higher)](databases/mongodb.md)
+- [MongoDB (version 3.6 or higher)](databases/mongodb.md) <!-- MongoDB supported version is from https://www.mongodb.com/support-policy -->
```

Comment:
3.4 was EOL'ed over a year ago (4.4 is the current version), and we can really generate much more efficient queries using some 3.6+ features, so it's time to bump our min supported version